TWLight/emails/templates/emails/access_code_email-body-text.html

```diff
@@ -1,9 +1,17 @@
 {% load i18n %}
 {% comment %}Translators: This email is sent to users with their access code. Don't translate Jinja variables in curly braces like {{ user }} or {{ partner }}. Translate Wikipedia Library in the same way as the global branch is named (click through from https://meta.wikimedia.org/wiki/The_Wikipedia_Library).{% endcomment %}
 {% blocktrans trimmed %}
+Dear {{ editor_wp_username }},
+
 Your approved application to {{ partner }} has now been finalised. Your access code or login details can be found below:
 
-{{ access_code }}
+{{ access_code }}.
+
+User instructions are {{user_instructions}}.
```

Comment:
@Samwalton9 Do you think this could just be `{{user_instructions}}`? `User instructions are` is going to an awkward fit with some of the `{{user_instructions}}` and this wrapping text would also get sent to translatewiki, adding another layer of potential odd grammar and extra work.
@YashAgrawal9265, below is an example of how some of our `user_instructions` are written.
```
Log in with the credentials you'll receive shortly at https://example.com/. Upon logging in for the first time, please take an example action. You are welcome to take another example action.
```